### PR TITLE
Added questions in CfU  javascript-4

### PR DESCRIPTION
**Pull request by raibove**
Added questions in check for understanding under the topic javascript-4

Fixed issue #700

### CODE REVIEW

File: javascript/javascript-4.md [hunk 1]
```diff
@@ -88,3 +89,8 @@ The output should look similar to this:
 `Erin is willing to go to Banff, Alberta, Canada or Iceland.`  
 `Janet is willing to go to London or Hogwarts.`  
 
+### Check for Understanding
+
+- What are the different types of loops in JavaScript?
+- Differentiate between for loop and while loop?
```
Comment (Kundan28) [inline javascript/javascript-4.md]: You can also mention do-while loop.

File: javascript/javascript-4.md [hunk 2]
```diff
@@ -88,3 +89,8 @@ The output should look similar to this:
...
+- Write a program in JavaScript to print Multiplication table of given number using for loop? Also try the program using while loop.
```
Comment (Kundan28) [inline javascript/javascript-4.md]: Why is there a question mark ?

File: javascript/javascript-4.md (hunk 1 above)
Comment (alodahl) [reply javascript/javascript-4.md]: Kundan, is do-while covered in this lesson? No new info should be covered in check for understanding.

Comment (alodahl) [reply javascript/javascript-4.md]: Also, line 95 is not a question @raibove

Comment (Kundan28) [reply javascript/javascript-4.md]: But then the meaning of the first question goes wrong...as there are 3 types of loops in JavaScript...

Comment (alodahl) [reply javascript/javascript-4.md]: Can you check if do-while is in the lesson?   If not, she should rephrase line 94.  If it is, she should follow your first comment here.

Comment (Kundan28) [reply javascript/javascript-4.md]: I have checked, do-while loop is not there in the lesson.

Comment (raibove) [reply javascript/javascript-4.md]: @alodahl  @Kundan28  I have changed the questions Please review the new questions.

**Comment by raibove** [comment]
@alodahl  @Kundan28 , Are the questions correct?

File: javascript/javascript-4.md [hunk 3]
```diff
@@ -88,3 +89,18 @@ The output should look similar to this:
 `Erin is willing to go to Banff, Alberta, Canada or Iceland.`  
 `Janet is willing to go to London or Hogwarts.`  
 
+### Check for Understanding
+
+- Differentiate between for loop and while loop?
+- Write down the syntax of Nested for loop in JavaScript?
```
Comment (alodahl) [inline javascript/javascript-4.md]: Write down the syntax of **a** Nested for loop in JavaScript**.**

File: javascript/javascript-4.md [hunk 4]
```diff
@@ -88,3 +89,18 @@ The output should look similar to this:
...
+- Write a program in JavaScript to print Multiplication table of given number using for loop. Also try the program using while loop.
```
Comment (alodahl) [inline javascript/javascript-4.md]: Can you move this to line 91 and make it:
### Challenge
- Write a program in JavaScript to print Multiplication table of given number using for loop. Also try the program using while loop.

**Pull request merged**